charts/opensearch/values.yaml

```diff
@@ -131,6 +131,9 @@ image:
 
 podAnnotations: {}
   # iam.amazonaws.com/role: es-cluster
+  
+# Deployment annotations
+annotations: {}
```

Comment:
Hey Can we please use as `openSearchAnnotations`?, also please change the comment to `# OpenSearch Statefulset annotations`
Thank you

Reply:
Changed to openSearchAnnotations & comments to stateful set annotations